packages/sanity/src/core/releases/__fixtures__/release.fixture.ts

```diff
@@ -0,0 +1,17 @@
+import {type ReleaseDocument} from '../store/types'
+
+export const activeScheduledRelease: ReleaseDocument = {
```

Comment:
Thinking that we'd continue to create others here eg `activeASAPRelease` as are necessary for test cases across our suite. All fixtures would be correct and valid states, eg `activeASAPRelease` would not have an `intendedPublishAt` since that is not a valid state machine value